### DISCUSSION TITLE
Provider: ORCID and/or ROR

### DISCUSSION THREAD

**Discussion by jlaura** [radiantearth/stac-spec#1229] [General]
I wonder if consider might be given to extending the provider object to include [ORCID](https://orcid.org/) or [RoR](https://ror.org/)? For academic and governmental data producers or data providers, these identifier can be as important as a DOI. 

If this idea has some traction, discussion would be valuable about whether this would be an addition to the provider, an addition to an extension like [scientific citation](https://github.com/stac-extensions/scientific), or an entirely new extension. I have not open this discussion on a particular extension because this relates directly to the provider object that is in the core specification.

**Comment by m-mohr** [comment]
As DOI is in scientific, I could see this bein in the scientific extension, too. sci:orcid and sci:ror for example, which then can be used in various places, including the Provider Object.

**Reply by jlaura** [reply]
The DOI describes publications and/or the data themselves. I agree having them in the sci extension makes sense. These identifiers are unique to the provider and only maintained at the collection level.  It does not make sense to me to split the unique identifier for the provider into an extension. 

In an FGDC metadata file the provider includes things like email address. If one wanted to also provide contact information for providers, where would that go? Surely not the science extension, but into the contacts extension. Now this is fragmenting what is a logic grouping for information (name, licensor, contact information, organizational unique identifier) across the main collection spec and 2+ extensions. What is being gained by this decomposition of information for what I perceive as a lot of added cognitive load on anyone trying to create or consume provider information?

**Comment by m-mohr** [comment]
See also https://github.com/stac-extensions/scientific/issues/10
